patches/tModLoader/Terraria/GameContent/UI/States/UIWorkshopHub.cs.patch

```diff
@@ -55,33 +55,28 @@
  			uIElement3 = MakeButton_OpenPublishWorldsMenu();
  			uIElement3.HAlign = 0f;
  			uIElement3.VAlign = 1f;
-@@ -98,6 +_,30 @@
+@@ -98,6 +_,25 @@
  			uIElement3.HAlign = 1f;
  			uIElement3.VAlign = 1f;
  			uIElement2.Append(uIElement3);
 +			*/
 +
 +			// Added by TML [
 +			var modsMenu = MakeButton_OpenModsMenu();
-+			modsMenu.HAlign = 0f;
++			modsMenu.Width.Set(-10f, 1f);
```

Comment:
Try this with `modsMenu.Width.Set(0f, 1f);`, it might fix the weird spaces on left and right